registries/_format/email.md

```diff
@@ -0,0 +1,22 @@
+---
+owner: baywet
+issue:
+description: An email address as defined as Mailbox in RFC5351
```

Comment:
Sure this is the correct RFC? 

https://www.rfc-editor.org/rfc/rfc5351.html is "An Overview of Reliable Server Pooling Protocols".

Maybe you meant https://www.rfc-editor.org/rfc/rfc5321 "Simple Mail Transfer Protocol"
```suggestion
description: An email address as defined as Mailbox in RFC5321
```